multidict/_compat.py

```diff
@@ -10,5 +10,6 @@
 if USE_EXTENSIONS:
     try:
         from . import _multidict  # type: ignore[attr-defined]  # noqa: F401
-    except ImportError:
+    except ImportError:  # pragma: no branch
```

Comment:
Looks like this doesn't work for some reason. Maybe, I was wrong, and it does indeed need a “no cover”. This has to be checked because I was commenting from mobile earlier w/ no way to verify. I suppose coveragepy might not treat this as a branch, after all.

Reply:
updated